src/main/java/appeng/container/implementations/ContainerInterfaceTerminal.java

```diff
@@ -206,12 +205,10 @@ public void doAction( final EntityPlayerMP player, final InventoryAction action,
 			final ItemStack is = inv.server.getStackInSlot( slot );
 			final boolean hasItemInHand = !player.inventory.getItemStack().isEmpty();
 
-			final InventoryAdaptor playerHand = new AdaptorPlayerHand( player );
-
-			final WrapperInvSlot slotInv = new PatternInvSlot( inv.server );
+			final InventoryAdaptor playerHand = InventoryAdaptor.getAdaptor( player );
 
-			final IInventory theSlot = slotInv.getWrapper( slot );
-			final InventoryAdaptor interfaceSlot = new AdaptorIInventory( theSlot );
+			final IItemHandlerModifiable theSlot = new PatternSlotWrapper(inv.server, slot);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') '(' is not followed by whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') ')' is not preceded with whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.whitespace.ParenPadCheck)